src/lang.js

```diff
@@ -367,6 +367,9 @@
 
             var leftDelimiter = matches[1];
             var leftNumber = convertNumber(matches[2]);
+            if (leftNumber === Infinity) {
```

Comment:
Hey @nanaya ! Can you give me a hint on this condition? The rest looks good to me :)